docker-compose.yml

```diff
@@ -3,6 +3,30 @@ version: "3.8"
 # TODO: Containerise frontend as well
 
 services:
+  web:
+    build: ./frontend/charity-management-app
+    ports:
+      - 5173:5173
+    volumes: 
+      - ./frontend/charity-management-app:/app
+      - /app/node_modules
```

Comment:
We don't actually need these volumes because we are using docker compose watch. The volumes are used to mount our local filesystem to the docker container, but because docker watch will watch for changes and update the docker container, we can get rid of the volumes.

Here is a link to the finished docker-compose.yml from that javascript mastery video I sent. 
https://github.com/adrianhajdin/docker-course/blob/main/mern-docker/compose.yml
Note that the web service doesn't have any volumes.